conda.recipe/meta.yaml

```diff
@@ -1,34 +1,74 @@
 {% set data = load_setup_py_data() %}
 
 package:
-    name: weldx
-    version: {{ data["version"] }}
+  name: weldx
+  version: {{ data["version"] }}
 
 source:
     path: ..
 
 build:
-  script: pip install . -v
   number: 0
   noarch: python
+  script: pip install . -v
 
 requirements:
-    host:
-        - python >=3.8
-        - pip
-        - setuptools_scm
-    run:
-        - python >=3.8
-        # dependencies are defined in setup.py
-        {% for dep in data['install_requires'] %}
-        - {{ dep.lower() }}
-        {% endfor %}
+  host:
+    - pip
+    - setuptools_scm
+    - python >=3.8
+  run:
+    - python >=3.8
+    # dependencies are defined in setup.py
+    {% for dep in data['install_requires'] %}
+    - {{ dep.lower() }}
+    {% endfor %}
 
 test:
-    imports:
-        - weldx
-        - weldx.visualization
+  requires:
+    - pytest
+    - pytest-xdist
+  source_files:
+    - tests/*.py
```

Comment:
Does passing "tests/" here not work?

Why do you need to pass the icons separately? How are they used in the tests?

Reply:
they get serialized as external files in a part of the asdf test suite

https://github.com/BAMWelDX/weldx/blob/bd643850b670a9da087e7739406bc7074a0dcc84/tests/asdf_tests/test_asdf_core.py#L416-L430